buildSrc/src/main/groovy/io.micronaut.build.internal.test-fixtures.gradle

```diff
@@ -0,0 +1,3 @@
+plugins {
+    id 'java-test-fixtures'
```

Comment:
I am not sure it is worth creating a new file to apply a single plugin, we could just apply the plugin in `gcp-common/build.gradle`

Reply:
I'm ambivalent about that. This is a result of following the way it is done in `micronaut-test-resources` as I was initially having some odd compilation issues when adding the plugin and following what was done there got me past that. That turned out to be more an issue with how dependencies are managed for test-fixtures so I'm fine with getting rid of this file and moving the plugin configuration.